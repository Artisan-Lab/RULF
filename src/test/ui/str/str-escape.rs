// check-pass
fn main() {
    let s = "\

             ";
    //~^^^ WARNING multiple lines skipped by escaped newline
    let s = "foo\
  Â           bar
             ";
    //~^^^ WARNING non-ASCII whitespace symbol '\u{a0}' is not skipped
}
