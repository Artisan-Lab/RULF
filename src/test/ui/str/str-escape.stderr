warning: multiple lines skipped by escaped newline
  --> $DIR/str-escape.rs:3:14
   |
LL |       let s = "\
   |  ______________^
LL | |
LL | |              ";
   | |_____________^ skipping everything up to and including this point

warning: non-ASCII whitespace symbol '\u{a0}' is not skipped
  --> $DIR/str-escape.rs:7:17
   |
LL |       let s = "foo\
   |  _________________^
LL | |   Â           bar
   | |   ^ non-ASCII whitespace symbol '\u{a0}' is not skipped
   | |___|
   | 

warning: 2 warnings emitted

